### PR TITLE
Added HEALTHCHECK for SOA docker image and renamed image name 

### PR DESCRIPTION
**Pull request by sbattagi**
Hi,

This PR contains the fixes for:

1. Added HEALTHCHECK support for SOA docker images.
2. Fixed code in SOA Dockerfile to apply OPATCH patch.
3. Updated SOA image name from oracle/soa:$VERSION  to oracle/soasuite:$VERSION.

Also updated the documentation for these changes. 
Please review and merge this request.

Thanks
Sambasiva

### CODE REVIEW

**Comment by Djelibeybi** [comment]
This is waiting on a review by @rdas0405 as code owner before it can be merged.

**Review by rdas0405 — APPROVED** [review]

**Pull request merged**
